test/webpack-integration.test.js

```diff
@@ -46,3 +102,27 @@ function readFileOrEmpty(path) {
     return '';
   }
 }
+
+const defaultConfig = {
```

Comment:
I think because of babel this gets transpiled to `var` and hoisting works fine, but _technically_ this wouldn't work. Maybe add it to a separate file? Or at least to the top of the file. Also, I'd suggest just outputting to /js like the others and simplify the .gitignore file...

Reply:
Good Call